SpatialGDK/Source/SpatialGDKEditor/Private/SpatialTestSettings.cpp

```diff
@@ -62,13 +70,15 @@ void FSpatialTestSettings::Override(const FString& MapName)
 	}
 
 	// Generated config, applied to generated maps
-	FString GeneratedMapOverridesFilename =
-		GeneratedOverrideSettingsBaseFilename + FPackageName::GetShortName(MapName) + (OverrideSettingsFileExtension);
-	if (FPaths::FileExists(GeneratedMapOverridesFilename))
+	FString GeneratedMapConfigurationFilename;
+	if (GenerateMapConfigurationFilename(MapName, GeneratedMapConfigurationFilename))
```

Comment:
Why this change? I think this function is pretty opaque?

Reply:
Ok I see its used twice, I think I'd prefer a simple static function that returns a generated name and an explicit `FPaths::FileExists` call, I think its the same amount of lines but clearer intent?

Reply:
Yeah I agree, this is definitely clearer, will do!